test/runtimeconfig.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "crictl info prints out the crio runtime config" {
+	start_crio
+	info_json=$(crictl info -o json)
+
+	run -0 jq -e '.config.crio' <<< "$info_json"
+
+	default_runtime=$(jq -r ".config.crio.DefaultRuntime" <<< "$info_json")
+	expected=$(get_runtime)
+	[[ "${default_runtime}" == "$expected" ]]
+
+	run -0 jq -e ".config.crio.Runtimes.${default_runtime}" <<< "$info_json"
+
+	runtime_type=$(jq -r ".config.crio.Runtimes.${default_runtime}.RuntimeType" <<< "$info_json")
+	[[ "${runtime_type}" == "oci" ]]
+
+	runtime_binary_path=$(jq -r ".config.crio.Runtimes.${default_runtime}.RuntimePath" <<< "$info_json")
+	# shellcheck disable=SC2153
+	expected=$RUNTIME_BINARY_PATH
+	[[ "${runtime_binary_path}" == "$expected" ]]
```

Comment:
You don't have to suppress the lint by writing this way, I guess.

```suggestion
	[[ "${runtime_binary_path}" == "$RUNTIME_BINARY_PATH" ]]
```

Reply:
That didn't work unfortunately :(

Reply:
It worked. The test failed for a different reason.

Reply:
oh, let me check again then

Reply:
This is the error I am now seeing
```
In test/runtimeconfig.bats line 26:
	[[ "${runtime_binary_path}" == "$RUNTIME_BINARY_PATH" ]]
                                        ^------------------^ SC[21](https://github.com/cri-o/cri-o/actions/runs/14522092308/job/40745203724?pr=9129#step:4:22)53 (info): Possible misspelling: RUNTIME_BINARY_PATH may not be assigned. Did you mean runtime_binary_path?
```

Reply:
@bitoku Please let me know If I need to add disable check back

Reply:
Actually, I take back what I said. I just needed to change the variable name so shellcheck stops flagging the "possible typo" issue

Thanks @bitoku! I'll resolve this comment